app/views/health/sdh_case_management_notes/edit.haml

```diff
@@ -18,3 +19,40 @@
         }
       });
     });
+
+    // Show type in box for other values
+    $('body').on('change', '.string-collection-with-other__control', function(e) {
```

Comment:
This is the same as line 44, so I think you can collapse the two into one `on` hook.

Reply:
Yes, it is the same, and in the original code they were together -- it just seemed clearer to me, to have two smaller methods that handle the two different cases. If you would prefer one `on` hook, I move it into 2 functions called from a single hook?

Reply:
I think one hook is better, I worry about duplicate hooks touching the same stuff, it means potentially calling the same thing twice, which can cause issues down the line.